Week-7/Assignment7.6.js

```diff
@@ -0,0 +1,56 @@
+class Queue{
+	
+constructor()
+{
+	this.s1 = [];
+	this.s2 = [];
+}
+
```

Comment:
can you implement this using constructor function and not class?

Reply:
Done